spec.bs

```diff
@@ -2792,6 +2843,69 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+<h4 id=fencedtreeclick-header><code>fencedtreeclick</code> event type</h4>
```

Comment:
Let's link to the dfn of `fencedtreeclick` here.

Reply:
Done